### PR TITLE
[3.3] Reduce README to bare minimum, and point to official documentation

### PR DESCRIPTION
**Pull request by GawainLynch**
As discussed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,81 +13,41 @@ all, it's a joy to use!
...
-[required-libraries]: https://docs.bolt.cm/other/credits#used-libraries-components
 [MIT-license]: http://opensource.org/licenses/mit-license.php
+[docs]: https://docs.bolt.cm/installation
+[support]: https://bolt.cm/community
 [travis]: http://travis-ci.org/bolt/bolt
 [travis-badge]: https://travis-ci.org/GawainLynch/bolt.svg?branch=release%2F3.3
 [codeclimate]: https://lima.codeclimate.com/github/bolt/bolt
 [codeclimate-badge]: https://lima.codeclimate.com/github/bolt/bolt/badges/gpa.svg
 [sensio-insight]: https://insight.sensiolabs.com/projects/4d1713e3-be44-4c2e-ad92-35f65eee6bd5
 [sensio-badge]: https://insight.sensiolabs.com/projects/4d1713e3-be44-4c2e-ad92-35f65eee6bd5/mini.png
```
Comment (bobdenotter) [inline README.md]: … until this can be removed

File: README.md [hunk 2]
```diff
@@ -13,81 +13,41 @@ all, it's a joy to use!
...
-[bolt-forum]: https://discuss.bolt.cm
-[bolt-twitter]: https://twitter.com/boltcm
-[silex]: http://silex.sensiolabs.org
-[symfony]: http://symfony.com
-[sf-components]: http://symfony.com/components
-[required-libraries]: https://docs.bolt.cm/other/credits#used-libraries-components
 [MIT-license]: http://opensource.org/licenses/mit-license.php
+[docs]: https://docs.bolt.cm/installation
+[support]: https://bolt.cm/community
 [travis]: http://travis-ci.org/bolt/bolt
```
Comment (bobdenotter) [inline README.md]: This …

File: README.md (hunk 1 above)
Comment (GawainLynch) [inline README.md]: They are still used though

Comment (bobdenotter) [inline README.md]: Derp! Nevermind me! Nothing to see here.

Comment (GawainLynch) [inline README.md]: Phew … More context in my comment might have helped my avoid a shock then when you pressed the merge button and the notification popped up on my screen … as my comment was intended to mean "Did I miss something obvious" :rofl: 

Lets just rack this up to #teamwork :smile:

**Pull request merged**